test/mpi/Makefile.mtest

```diff
@@ -20,15 +20,15 @@ LDADD = $(top_builddir)/util/mtest.$(OBJEXT) $(top_builddir)/util/mtest_common.$
 
 # Add libdtpools support
 AM_CPPFLAGS += -I$(top_srcdir)/dtpools/include
-LDADD += $(top_builddir)/dtpools/src/.libs/libdtpools.a
+LDADD += $(top_builddir)/dtpools/src/libdtpools.la
 
 $(top_builddir)/util/mtest.$(OBJEXT): $(top_srcdir)/util/mtest.c
 	(cd $(top_builddir)/util && $(MAKE) mtest.$(OBJEXT))
 
 $(top_builddir)/util/mtest_common.$(OBJEXT): $(top_srcdir)/util/mtest_common.c
 	(cd $(top_builddir)/util && $(MAKE) mtest_common.$(OBJEXT))
 
-$(top_builddir)/dtpools/src/.libs/libdtpools.a: $(top_srcdir)/dtpools/src/*.c
+$(top_builddir)/dtpools/src/libdtpools.la:
```

Comment:
@pavanbalaji I had to leave this or else `make testing` won't be able to trigger `dtpools` compilation. Making this work more seemingly with `libtool` will require more work on the testsuite (which includes the conversion of `mtest` into a convenience library). As @hzhou has suggested, this should be a separate PR thus I am not addressing this here as I was initially trying to do.